.cirrus.yml

```diff
@@ -268,6 +268,18 @@ ubuntu24_task:
     << : *RESOURCES_TEMPLATE
   << : *CI_TEMPLATE
 
+# Same as above, but using Clang and libc++
+ubuntu24_clang_libcpp_task:
+  container:
+    # Ubuntu 24.04 EOL: Jun 2029
+    dockerfile: ci/ubuntu-24.04/Dockerfile
+    << : *RESOURCES_TEMPLATE
+  << : *CI_TEMPLATE
```

Comment:
I'm going to add `SKIP_TASK_ON_PR` - we'll still start to get errors/failures during nightly builds which should be good enough.